src/cleanfigure.m

```diff
@@ -322,6 +345,152 @@ function pruneOutsideBox(meta, handle)
         0.0 < lambda(:, 2) & lambda(:, 2) < 1.0;
 end
 % =========================================================================
+function [xData, yData] = getVisibleData(meta, handle)
```

Comment:
The naming should be `Visual` or something else, since the function applies transformations, i.e. log scaling and projections. `Visible` refers to a dicotomic state (on/off) and is also a very common property, hence the source of potential confusion. Moreover, the confusion is especially apparent next to the function `segmentVisible()` which actually checks if datapoint are visible or not.